Lib/asyncio/streams.py

```diff
@@ -70,6 +70,14 @@ def connect(host=None, port=None, *,
             server_hostname=None,
             ssl_handshake_timeout=None,
             happy_eyeballs_delay=None, interleave=None):
+    """Connect to TCP socket on *host* : *port* address and return a `Stream`
```

Comment:
Minor: in docstrings for CPython we usually avoid markup. The docstring is for human reading, the documentation is generated from hand-written Sphinx pages.
`help()` doesn't render embedded markup too.
I know, older docstrings for the module contains back tickets at least.
Should we add new markup in this PR or avoid it in new updates?
@1st1 I need your opinion here

Reply:
Using `*` for parameters improves readability IMO, using backticks isn't necessary (again, IMO).

Reply:
Ok, I have no strong opinion here.

Reply:
I have kept them and some of the other docstrings also have backticks to represent class names.